charts/alfresco-api-gateway/templates/role.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Chart.Name }}
+rules:
+- apiGroups:
+  - extensions
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
```

Comment:
This may just be boilerplate, but do we need more than `get`, `list`, and `watch` for this role?